src/website/views/components/data-table/_examples.pug

```diff
@@ -0,0 +1,846 @@
+//-             div Name 2
+//-           .chi-data-grid__column(data-label="Column 2")
+//-             div name-2
+//-           .chi-data-grid__column(data-label="Column 3")
+//-             div Dec 18, 2020 3:26 PM
+//-         .chi-data-grid__row
+//-           .chi-data-grid__column.-lh--1
+//-             input.chi-checkbox__input(type="checkbox", id=`chk-4`)
+//-             label.chi-checkbox__label(for=`chk-4`)
+//-               .-sr--only Label
+//-           .chi-data-grid__column(data-label="Column 1")
+//-             div Name 3
+//-           .chi-data-grid__column(data-label="Column 2")
+//-             div name-3
+//-           .chi-data-grid__column(data-label="Column 3")
+//-             div Dec 18, 2020 3:26 PM
+//- 
+//-   .example-tabs.-pl--2
+//-     ul.chi-tabs#example-base
+//-       li.-disabled
+//-         a(href=`#webcomponent-base`) Web Component
+//-       li.-active
+//-         a(href=`#html-base`) HTML Blueprint
+//-   .chi-tabs-panel#webcomponent-base
+//-   .chi-tabs-panel.-active#html-base
+//-     :code(lang='html')
+//-       <div class="chi-data-grid">
+//-       </div>
+//- 
+//- h3 Example 1 -- 2 columns mobile
+//- .example.-mb--3
+//-   .-p--3.-p-md--5
+//-     .chi-data-grid.-two-columns-mobile
+//-       .chi-data-grid__head
+//-         .chi-data-grid__row
+//-             .chi-data-grid__column
+//-                span Column1
+//-             .chi-data-grid__column
+//-               span Column2
+//-       .chi-data-grid__body
+//-         .chi-data-grid__row
+//-           .chi-data-grid__column(data-label="Column 1")
+//-             div Row 1
+//-           .chi-data-grid__column(data-label="Column 2")
+//-             div Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc dui magna, finibus id tortor sed, aliquet bibendum augue.
+//-         .chi-data-grid__row
+//-           .chi-data-grid__column(data-label="Column 1")
+//-             div Row 1
+//-           .chi-data-grid__column(data-label="Column 2")
+//-             div Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc dui magna, finibus id tortor sed, aliquet bibendum augue.
+//- 
+//-   .example-tabs.-pl--2
+//-     ul.chi-tabs#example-base
+//-       li.-disabled
+//-         a(href=`#webcomponent-base`) Web Component
+//-       li.-active
+//-         a(href=`#html-base`) HTML Blueprint
+//-   .chi-tabs-panel#webcomponent-base
+//-   .chi-tabs-panel.-active#html-base
+//-     :code(lang='html')
+//-       <div class="chi-data-grid">
+//-       </div>
+
+h3 Base
+.example.-mb--3
+  .-p--3.-p-md--5
+    .chi-data-table
+      .chi-data-table__head
+        .chi-data-table__row
+            .chi-data-table__cell
+               span Name
+            .chi-data-table__cell
+              span Email
+            .chi-data-table__cell
+             span Phone Number
+            .chi-data-table__cell
+              span Address
+            .chi-data-table__cell
+              span Last login
```

Comment:
Casing is inconsistent in every example. Can you do a find and replace and change all instances of 'Last login' to 'Last Login'